packages/front-end/components/Metrics/MetricForm.tsx

```diff
@@ -816,6 +825,43 @@ GROUP BY
             green, yellow, or red.
           </small>
         </div>
+        <div className="form-group">
+          Minimum threshold for significance
+          <input
+            type="number"
+            className="form-control"
+            {...inputs.minThresholdSignificance}
+          />
+          <small className="text-muted">
+            The number of events this metric requires before this metric will
+            show as significant. (default{" "}
+            {defaultMinConversionThresholdSignificance})
+          </small>
+        </div>
+        <div className="form-group">
+          Max variance percent
+          <div className="col px-0">
+            <span
+              style={{
+                position: "absolute",
+                right: "4px",
+                top: "6px",
+                color: "#888",
+              }}
+            >
+              %
+            </span>
+            <input
+              type="number"
+              className="form-control"
+              {...inputs.varianceThreshold}
+            />
+          </div>
+          <small className="text-muted">
+            Results with variance above this percentage will be flagged as
+            suspicious. (default {defaultVarianceThreshold * 100})
```

Comment:
For this copy, I would do

**Max Percent Change**
An experiment that changes the metric by more than this percent will be flagged as suspicious (default X%)